src/spac/utils.py

```diff
@@ -70,3 +71,229 @@ def regex_search(pattern, str):
     all_results = list(all_results)
 
     return all_results
+
+
+def check_list_in_list(

```

Comment:
Just curious why this is part of this commit? I thought we already merged this part.

Reply:
Ahh sorry the merge should be to dev not main, I will change this. Once it is to dev, this should not appear.